test/lint/lint-assertions.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2018-2022 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+#
+# Check for assertions with obvious side effects.
+
+import sys
+import subprocess
+
+
+def exec_cmd(command: [], error_msg: ""):
```

Comment:
Would prefer not to have this wrapper and use `subprocess.check_output` directly below where relevant. The amount of shared code is minimal, no need to define a function.

Reply:
I did it because of the try catch needed to check the return code when it is empty.

Reply:
Fair enough. Though this logic is specific to `git grep`. It makes sense to make it a  `git_grep` wrapper specifically.